test/end-to-end.js

```diff
@@ -163,6 +173,7 @@ describe('end-to-end', function() {
       };
       boot(app, bootOptions, function(err) {
         if (err) return done(err);
+        app.use(app.get('restApiRoot'), app.loopback.rest());
```

Comment:
Please remove, the `REST `middleware must be added by `loopback-boot`. If it is not, then skip the test for now.